library/src/containers/Schemas/Schema.tsx

```diff
@@ -74,8 +76,22 @@ export const SchemaComponent: React.FunctionComponent<Props> = ({
       <div className={`${bemClasses.element(`${className}-table`)} p-4`}>
         {renderSchemaProps(name, schema)}
       </div>
-      {/* we need to disable this component if schema has "not" field anywhere in it */}
-      {hasNotField ? null : (
+
+      {examples ? (
+        (() => {
+          const allExamples: object[] = [];
+          const examplesLength = examples.length;
+          examples.forEach((el, i) => {
+            const title =
+              examplesLength > 1 ? `${exampleTitle} ${i}` : exampleTitle;
+            allExamples.push(
+              <SchemaExampleComponent title={title} example={el} />,
+            );
+          });
+          return allExamples;
+        })()
+      ) : // we need to disable this component if schema has "not" field anywhere in it
```

Comment:
You can change this lines to simpler code:

```ts
{examples ? examples.map((el, i) => (
   <SchemaExampleComponent title={examples.length > 1 ? `${exampleTitle} ${i}` : exampleTitle} example={el} />,
)) : hasNotField ...}
```

You can also make for `examples.length` additional variable before return statement `const examplesLength = examples.length`.